ME/Controller/BuyFailedViewController.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  buyFailedViewController.swift
+//  ME
+//
+//  Created by 새미 on 2023/09/10.
+//
+
+import UIKit
+import SnapKit
+
+final class BuyFailedViewController: UIViewController {
+    
+    let popupView = BuyFailedPopupView()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        setup()
+    }
+    
+    // MARK: - Setting
+    private func setup() {
+        self.view.backgroundColor = .black.withAlphaComponent(0.6)
+        setLayout()
+        actionButton()
+    }
+    
+    // MARK: - Layout
+    private func setLayout() {
+        view.addSubview(popupView)
+        
+        popupView.snp.makeConstraints {
+            $0.centerX.centerY.equalToSuperview()
+            $0.width.equalTo(278)
+            $0.height.equalTo(156)
+        }
+    }
+    
+    // MARK: - CheckButton Action
+    func actionButton() {
+        popupView.checkButton.addTarget(self, action: #selector(actionDismiss), for: .touchUpInside)
+    }
+    
+    @objc func actionDismiss() {
+        dismiss(animated: false)
+    }
+    
+}
+
+    
```

Comment:
```suggestion
    }
}
    
```
불필요한 여백은 줄여보는 게 어떨까?
코드 컨벤션에 대해서도 좀 더 생각해보면 좋을 것 같아.

지금 남긴 부분 말고도 전체적으로 불필요한 공백들이 보이는데 같이 신경써서 수정해보자~